### ISSUE TITLE
Client crashes periodically

### ISSUE THREAD

**Issue by k0a1a** [nextcloud/android#7121]
### Steps to reproduce
1. have three accounts enabled, some of which have large number of files, one account has auto-upload enabled
2. connect to several calendars
3. leave running in the background

### Expected behaviour
The app used not to crash 

### Actual behaviour
Crashes with the trace below

### Can you reproduce this problem on https://try.nextcloud.com?

didn't try that yet
### Environment data
Android version: 9

Device model: Samsung s9plus

Stock or customized system: rooted stock 

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/firebase/analytics/connector/AnalyticsConnector;
	at com.google.firebase.messaging.zzo.zza(com.google.firebase:firebase-messaging@@20.1.3:120)
	at com.google.firebase.messaging.zzo.zza(com.google.firebase:firebase-messaging@@20.1.3:1)
	at com.google.firebase.messaging.FirebaseMessagingService.zzc(com.google.firebase:firebase-messaging@@20.1.3:50)
	at com.google.firebase.messaging.zze.run(com.google.firebase:firebase-messaging@@20.1.3:2)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at com.google.android.gms.common.util.concurrent.zza.run(Unknown Source:6)
	at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.firebase.analytics.connector.AnalyticsConnector" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/com.nextcloud.client-8JmuYUZFTg3VcGOCi4Mqxw==/base.apk"],nativeLibraryDirectories=[/data/app/com.nextcloud.client-8JmuYUZFTg3VcGOCi4Mqxw==/lib/arm64, /data/app/com.nextcloud.client-8JmuYUZFTg3VcGOCi4Mqxw==/base.apk!/lib/arm64-v8a, /system/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 8 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30130190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: star2lte
Model: SM-G965F
Id: PPR1.180610.011
Product: star2ltexx

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: G965FXXU5CSF2

```

**Comment by AndyScherzinger** [comment]
Duplicate of #4675

**Comment by AndyScherzinger** [comment]
Fixed with upcoming 3.14.0
